_i18n/en.yml

```diff
@@ -801,7 +801,24 @@ roadmap:
   p2pool: P2Pool released
   haveno: Haveno
 
-
+  ringsize-sera-tryp: increase ring size to 100+ through Seraphis or Triptych
+  seraphis-jamtis: Seraphis & jamtis
+  ospead: OSPEAD
+  released-0-17-3-0-cli: CLI 0.17.3.0 released
+  released-0-17-3-0-gui: GUI 0.17.3.0 released
+  released-0-17-3-1-gui: GUI 0.17.3.1 released
+  released-0-17-3-2-gui: GUI 0.17.3.2 released
+  released-0-17-3-2-cli: CLI 0.17.3.2 released
+  released-0-18: CLI and GUI 0.18 "Flourine Fermi" released
+  hardfork-v15: Network upgrade v15 Hardfork
+  ringsize-11-16: Ringsize increase 11 -> 16
+  viewtags: Viewtags (upto -40% wallet sync time)
+  feechanges: Fee changes
+  released-0-18-1: CLI and GUI 0.18.1 "Flourine Fermi" point release
+  xmr-eth-atomicswaps: Monero <-> Ethereum Atomic Swaps
+  bp++: Bulletproofs++
+  lws-subaddress: LWS Subaddress support
```

Comment:
I don't think t his is a major milestone that needs to be added to the roadmap.

Reply:
(I assume this is about the LWS subaddress') removed in updated commit

Reply:
Maybe worth changing into the full `Light Wallet Server support`. It's not long and would be much more clear for most people.